### DISCUSSION TITLE
Anyway to make this work with ts import path alias?

### DISCUSSION THREAD

**Discussion by rairulyle** [Dschungelabenteuer/vite-plugin-entry-shaking#57] [Q&A]
Title. We are using NX workspace and uses TS import path alias but it seems to not work on this plugin. I have to make a resolve or directly have the path in the config to make it work.

Is there a better way to do this?

**Comment by Dschungelabenteuer** [comment]
Hey! I'm not familiar with Nx workspaces but I'm rather confident there should not be any differences with other regular environments regarding path aliases. This repo provides some examples and most of them are actually using path aliases. See the [simplest example](https://github.com/Dschungelabenteuer/vite-plugin-entry-shaking/tree/main/examples/simple) here:  [those](https://github.com/Dschungelabenteuer/vite-plugin-entry-shaking/blob/main/examples/simple/tsconfig.json#L22) are path aliases as defined for TypeScript, and [those](https://github.com/Dschungelabenteuer/vite-plugin-entry-shaking/blob/main/examples/simple/vite.config.ts#L18-L20) are path aliases as defined for Vite. Note that most people would usually rely on [vite-tsconfig-paths](https://www.npmjs.com/package/vite-tsconfig-paths) to avoid the burden of maintaining two lists of the same aliases (both tsconfig and Vite config).

However, as I told before, I'm not familiar with Nx so I may be missing something. Could you please provide a simple repro if my above point wasn't helpful?
